packages/components/src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -171,7 +176,10 @@ export const Checkbox = ({
     const theme = useTheme();
 
     const handleKeyUp = (event: KeyboardEvent<HTMLElement>) => {
-        if (event.code === KEYBOARD_CODE.SPACE || event.code === KEYBOARD_CODE.ENTER) {
+        if (
+            isDisabled !== true &&
```

Comment:
I did this because in this component, there was possible that `isDisabled` could be undefined. I wasn't changing the original default values even when I noticed some inconsistencies regarding the components. Of course if we decide to have `isDisabled` set to `false` by default, it is definitely better to check against `false` here.

Reply:
I believe `isDisabled={undefined}` should behave like false, so let's have it more strict and use only boolean values.